.github/workflows/merge_pull_request.yml

```diff
@@ -12,6 +12,12 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Resolve possible conflicts ${{ github.event.client_payload.ReleaseBranchName }} with main
+      run: |
+        git checkout ${{ github.event.client_payload.ReleaseBranchName }}
+        git merge --no-edit --strategy-option=ours main
```

Comment:
Don't we need to push these changes?

Reply:
I believe you're right

Reply:
Please test this change on fork properly. It is committing to main branch and we need to be extra careful here.
I also believe push will fail if no new commits in master

Reply:
Done, thank you!